### PR TITLE
Added support for movies

### PR DESCRIPTION
**Pull request by SinTan1729**
This PR adds support for movies. The code is admittedly a derivative of the old tv show code, so there might be scope for deduplication and merging of functions in the future. But it works as expected.

### CODE REVIEW

**Comment by lukearran** [comment]
That's fantastic you've added movie support! :)

As you mentioned, if we can work on reducing code duplication then I'll be happy to merge this.

If you need any help with this, give me a shout and I'll find time in the next month to work with you on these changes.

Kindest Regards,

Luke

**Comment by SinTan1729** [comment]
I kinda forgot about this after I was done with my migration. I'll try to do some cleanup and deduplication in the coming weeks.

File: TimeToTrakt.py [hunk 1]
```diff
@@ -459,27 +468,356 @@ def processWatchedShows():
...
+                return moviesWithSameName[firstMatchSelectedIndex]
+            # Otherwise, return None, which will trigger the script to skip
+            # and move onto the next movie
+            else:
+                return None
+        # If the user has not provided a manual selection already in the process
+        # then prompt the user to make a selection
+        else:
+            print(
+                f"INFO - MANUAL INPUT REQUIRED: The TV Time data for Movie '{name}' has {len(moviesWithSameName)} matching Trakt movies with the same name."
```
Comment (WeirdAlex03) [inline TimeToTrakt.py:557]: For consistency with TV shows from PR #24, a bell output should also be added here

Comment (SinTan1729) [reply TimeToTrakt.py:557]: Sure, I'll do it.

**Comment by SinTan1729** [comment]
@lukearran I was trying to dedup and refactor the code a bit but whatever I do seems to further complicate it and decrease readability. The movies and show sections read from differently structured files and push to differently structured API objects. I could've tried to make a general object that'd abstract the calls for shows and movies but that also doesn't look nice.
In short, I don't really see a way to dedup it without using lot of if/else of cases, which decreases readability. Let me know what you think.

**Pull request merged**